tmail-backend/events/src/main/scala/com/linagora/tmail/events/TmailContactListener.scala

```diff
@@ -0,0 +1,33 @@
+package com.linagora.tmail.events
+
+import com.linagora.tmail.james.jmap.contact.EmailAddressContactSearchEngine
+import javax.inject.Inject
+import org.apache.james.events.EventListener.ReactiveGroupEventListener
+import org.apache.james.events.{Event, Group}
+import org.apache.james.jmap.api.model.AccountId
+import org.reactivestreams.Publisher
+import org.slf4j.{Logger, LoggerFactory}
+import reactor.core.scala.publisher.SMono
+
+class TmailContactListener @Inject()(contactSearchEngine: EmailAddressContactSearchEngine) extends ReactiveGroupEventListener {
+  private val LOGGER: Logger = LoggerFactory.getLogger(classOf[TmailContactListener])
+
+  override def getDefaultGroup: Group = PushListenerGroup()
+
+  override def reactiveEvent(event: Event): Publisher[Void] = {
+    event match {
+      // TODO, EmailAddressContactSearchEngine.index will be updated
+      case contactEvent: TmailContactUserAddedEvent => handlerContactUserAddedEvent(contactEvent)
```

Comment:
// TODO,
Need to rebase after https://github.com/linagora/tmail-backend/issues/362 already